bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat

```diff
@@ -0,0 +1,164 @@
+#***********************************************************************
+  PYTHIA8   = parton_shower
+  1.0       = shower_scale_factor ! multiply default shower starting  
+				  ! scale by this factor 
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses of all final state particles and partons. This *
+# can be changed in SubProcesses/set_scales.f)                         *
+#***********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.188   = muR_ref_fixed    ! fixed ren reference scale 
+ 91.118   = muF_ref_fixed    ! fixed fact reference scale
+# 91.188   = muF1_ref_fixed   ! fixed fact reference scale for pdf1
+# 91.188   = muF2_ref_fixed   ! fixed fact reference scale for pdf2
+#***********************************************************************
+# Renormalization and factorization scales (advanced and NLO options)  *
+#***********************************************************************
+# F        = fixed_QES_scale  ! if .true. use fixed Ellis-Sexton scale
+# 91.188   = QES_ref_fixed    ! fixed Ellis-Sexton reference scale
+  -1 = dynamical_scale_choice ! Choose one (or more) of the predefined
+ 		! dynamical choices. Can be a list; scale choices beyond the
+		! first are included via reweighting		 
+ 1.0  = muR_over_ref     ! ratio of current muR over reference muR
+ 1.0  = muF_over_ref  ! ratio of current muF over reference muF
+# 1        = muF1_over_ref    ! ratio of current muF1 over reference muF1
+# 1        = muF2_over_ref    ! ratio of current muF2 over reference muF2
+# 1        = QES_over_ref     ! ratio of current QES over reference QES
+#*********************************************************************** 
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+  1.0 = rw_rscale ! muR factors to be included by reweighting
+  1.0 = rw_fscale ! muF factors to be included by reweighting
+ True   = reweight_scale   ! reweight to get scale dependence
+#  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+#  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+#  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+#  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+  $DEFAULT_PDF_MEMBERS  = reweight_PDF     ! reweight to get PDF uncertainty
+ 292201   = PDF_set_min      ! First of the error PDF sets 
+ 292302   = PDF_set_max      ! Last of the error PDF sets
```

Comment:
292201   = PDF_set_min      ! First of the error PDF sets 
 292302   = PDF_set_max      ! Last of the error PDF sets

This should be removed